articles/quickstart/native/xamarin/01-login.md

```diff
@@ -52,36 +52,23 @@ Go to your [Application's Dashboard](${manage_url}/#/applications/${account.clie
 
 ## Integration
 
-To integrate Auth0 login into your application, simply instantiate an instance of the `Auth0Client` class, configuring the Auth0 Domain and Client ID, and also passing the Android Activity or View Controller from which you are executing the code:
+To integrate Auth0 login into your application, simply instantiate an instance of the `Auth0Client` class, configuring the Auth0 Domain and Client ID:
 
 ${snippet(meta.snippets.setup)}
 
-The steps for Logging the user in is different for Android and iOS. Note that this example is not using Xamarin Forms.
-
-### Android
-
-#### 1. Obtain the authorization URL
-
-Call the `PrepareLoginAsync` method which will return an `AuthorizeState` containing the authorization URL, state, nonce, and code challenge. You will need to store the `AuthorizeState` as it is required later on to process the redirect URL and exchange the authorization code for the tokens.
+You can then call the `LoginAsync` method which will redirect the user to the Auth0 Hosted Lock screen.
```

Comment:
The " Auth0 Hosted Lock screen" it's being called "the login screen" or "the universal login screen". We use the later when we need to explicitly differentiate it from Embedded Login, which is not the case.